homework/Thu_Pham/lesson_1/136-single-number.js

```diff
@@ -0,0 +1,18 @@
+//Solution for https://leetcode.com/problems/single-number/
+
+/*
+ * @param {number[]} nums
+ * @return {number}
+ */
+ var singleNumber = function(nums) {
+    const tmpArray = []
+    for(let i = 0; i < nums.length; i++) {
+        const index = tmpArray.findIndex((item) => item === nums[i])
```

Comment:
`findIndex` takes O(n) time => overall this approach will take O(n^2). In lesson 3, we will study hashmap, which can replace your tmpArray for a better runtime (O(1) for lookup).

For now, how about apply the XOR's property: x^x = 0, x^0=x